plugins/woo-ai/tsconfig.json

```diff
@@ -6,7 +6,8 @@
 	// Make sure we're not pulling types from external projects.
 		"typeRoots": [
 			"./typings",
-			"./node_modules/@types"
+			"./node_modules/@types",
+      "src/custom.d.ts",
```

Comment:
White space missing here before the array element 😅

Reply:
Nice catch!